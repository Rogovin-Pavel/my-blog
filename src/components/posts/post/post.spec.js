import React from 'react';
import renderer from 'react-test-renderer';
import Post from './post';
describe('Counter', () => {
  const post = {
          id: '1',
          title: 'post-title',
          content: 'post-content',
          date: new Date()
        },
        user = {
          id: '1',
          userName: 'Pavel'
        };
  
  test('snapshot renders', () => {
    const component = renderer.create(<Post post={post} user={user} onDeletePost={()=>{}}/>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});