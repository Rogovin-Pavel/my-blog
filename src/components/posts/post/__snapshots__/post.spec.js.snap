// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter snapshot renders 1`] = `
<div
  className="post col-12"
>
  <div
    className="post__header"
  >
    <div
      className="post__content"
    >
      <h4
        className="post__heading"
      >
        post-title
      </h4>
      <h6
        className="post__date"
      >
        12:55:02 GMT+0300 (GMT+03:00)
      </h6>
    </div>
    <div
      className="post__remove-btn"
      onClick={[Function]}
    >
      <img
        alt="remove button"
        className="post__remove-btn-image"
        src="remove-button.svg"
      />
    </div>
  </div>
  <div
    className="post__body"
  >
    <p
      className="post__paragraph"
    >
      post-content
    </p>
  </div>
</div>
`;
